it/connectors/CISDeductionsConnectorISpec.scala

```diff
@@ -43,6 +45,26 @@ class CISDeductionsConnectorISpec extends ConnectorIntegrationTest
   private val nino: String = "AA123123A"
   private val submissionId: String = "a111111a-abcd-111a-123a-11a1a111a1"
 
+  private val customerPeriodData = aGetPeriodData.copy(costOfMaterials = Some(100.00), grossAmountPaid = Some(100.00))
+  private val contractorPeriodData = customerPeriodData.copy(deductionFromDate = s"${taxYearEOY - 1}-04-06", deductionToDate = s"${taxYearEOY - 1}-05-05", submissionId = None, source = CONTRACTOR)
+  private val customerCisDeductions = aCISDeductions.copy(
```

Comment:
Is it possible to get rid of those variables?

Reply:
This was discussed with Roberto. It will need to be done in another chunk of refactoring work.